packages/ilios-common/addon/components/dashboard/calendar.js

```diff
@@ -18,15 +18,30 @@ export default class DashboardCalendarComponent extends Component {
   @service dataLoader;
   @service localeDays;
 
-  @tracked usersPrimarySchool;
-  @tracked absoluteIcsUri;
+  @storageFor('dashboard') dashboardStorage;
 
-  @tracked ourEvents = [];
+  get userContext() {
+    console.log(this.dashboardStorage.get('userContext'));
```

Comment:
extra log.
//I thought we had an eslint rule for this...

Reply:
urgl! i thought so too. fixed for now, follow-up PR incoming soon.

Reply:
:heavy_check_mark: